src/pages/Home.jsx

```diff
@@ -36,48 +68,76 @@ const Home = () => {
-              componentImageSrc="/assets/app-green.png"
-              componentImageAlt=""
-              title="An App for Caseworkers"
-              description="PASS allows users to quickly and securely share documents of their clients within the app. The app helps caseworkers verify and share documents such as ID and proof of income while retaining total control of them."
-            />
-            <HomeSection
-              isReallySmallScreen={isReallySmallScreen}
-              componentImageSrc="/assets/key-features-green.png"
-              componentImageAlt=""
-              title="Key Features"
-            />
-            <KeyFeatures
-              isReallySmallScreen={isReallySmallScreen}
-              icon={<SecurityIcon fontSize={isReallySmallScreen ? 'medium' : 'large'} />}
-              title="Secure Storage"
-              description="Store vital documents like IDs, Social Security information, birth certificates, medical records, and bank statements in a valid digital format."
-            />
-            <KeyFeatures
-              isReallySmallScreen={isReallySmallScreen}
-              icon={<Diversity1Icon fontSize={isReallySmallScreen ? 'medium' : 'large'} />}
-              title="Nonprofit-Caseworker Integration"
-              description="The platform facilitates smooth communication between nonprofit organizations, case workers, and the individuals they serve. It allows nonprofit organizations to maintain a contact list, and caseworkers are assigned contacts whose data they can access securely."
-            />
-            <KeyFeatures
-              isReallySmallScreen={isReallySmallScreen}
-              icon={<SupportIcon fontSize={isReallySmallScreen ? 'medium' : 'large'} />}
-              title="Support Service"
-              description="Verified documents can be used to facilitate access to service such as housing support and shelter accommodation. The platform simplifies the process of submitting necessary documents for such services."
-            />
+            <Box
+              height="calc(100vh - 64px)"
+              display="flex"
+              flexDirection="column"
+              justifyContent="center"
+              alignItems="center"
+            >
+              {logoSection}
+              <Typography
+                variant={isReallySmallScreen ? 'h4' : 'h3'}
+                component="p"
+                fontWeight="600"
+                mb={isReallySmallScreen ? 8 : 18}
+                color="primary"
+              >
+                Personal Access System for Services
+              </Typography>
+              <Typography color="secondary.main" sx={{ fontSize: '24px' }}>
+                Learn More
+              </Typography>
+              <IconButton aria-label="Scroll down" color="secondary" href="#about-pass">
+                <ArrowCircleDownOutlinedIcon sx={{ fontSize: '32px' }} />
+              </IconButton>
```

Comment:
Love the idea of a button to guide users along the site! Think it'll be nice to have smooth scrolling down as well if we can have that included within this PR.

I remember how to do this in CSS with a few lines. We just need to set `scroll-behavior: smooth` for the main `<html>` tag. Though in order to do that we'll need to create a CSS file again and import it to `index.jsx`.

https://github.com/codeforpdx/PASS/assets/14917816/3a4313bf-47d1-4d46-af9b-9d321b8f5323

The other way, of course, is to use the `useRef` hook to grab the element as a reference and make it run `scrollIntoView({ behavior: 'smooth' })` in React. A bit more complicated, but no CSS files required.

https://github.com/codeforpdx/PASS/assets/14917816/67d0b5b3-8828-4aa0-9f42-353fb6e30b46

Reply:
I'm not sure what branch you are showing there.  The down arrow icon button has smooth scrolled to the first `HomeSection` below since this commit on 11/16: 103db67cb0524799f6709d447b78ae237530f2de

If you are on branch `issue-456/home-page-structure-enhancements`, have you pulled down latest?

make sure you are NOT on `...enhancements--h1`

Reply:
Sorry, I could've sworn I implemented smooth scroll via a ref previously!!! 

I'm out of it. 

Pushed commit now.

Reply:
No problem, just checked the new commits now. They're up and running. 👍